tests/test-demo-buildsystem.sh

```diff
@@ -19,9 +19,12 @@
 
 set -euo pipefail
 
-if ! fusermount --version >/dev/null 2>&1; then
+if ! which fusermount >/dev/null 2>&1; then
     echo "1..0 # SKIP no fusermount"
     exit 0
+elif ! capsh --print | grep ^Current | grep -q cap_sys_admin; then
```

Comment:
Actually, from quickly testing it, it seems like your skip check for fuse should be enough. It seems like it's not enough to run the container with `--cap-add=SYS_ADMIN`, but it actually needs `--privileged` to have a writable `/etc/fuse`